### ISSUE TITLE
lucida ebuild

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [archenroot/gentoo-overlay#11]
Are you interested in this?
https://github.com/claritylab/lucida

I was writing the ebuild

**Comment by archenroot** [comment]
Definitely, anything related to AI I am interested having here. I work on object tracking and recognition in real time, but anything related to speech/text processing is in my interest.
Thanx.

**Comment by archenroot** [comment]
Good stuff, I never heard of this solution yet. Thanks again.
